### PR TITLE
Small documentation fixes

### PR DESCRIPTION
**Pull request by russjones**
* Updated Application Access documentation to reflect that it does not support gRPC proxying.
* Fixed profiling documentation where the `--debug` flag was omitted.

### CODE REVIEW

File: docs/pages/application-access/guides/api-access.mdx [hunk 1]
```diff
@@ -5,8 +5,9 @@ description: How to access REST APIs with Teleport Application Access.
 
 # Application API Access
 
-Teleport Application Access can be used to access applications' (REST or Teleport's own gRPC) APIs with
-tools like [curl](https://man7.org/linux/man-pages/man1/curl.1.html) or Postman.
+Teleport Application Access can be used to access HTTP applications' APIs with
```
Comment (zmb3) [inline docs/pages/application-access/guides/api-access.mdx:8]: gRPC would work with TCP app access, right?

**Review by alexfornuto — APPROVED** [review]
Copy looks good, pending Zac's query.

File: docs/pages/application-access/guides/api-access.mdx (hunk 1 above)
Comment (russjones) [reply docs/pages/application-access/guides/api-access.mdx:8]: Yeah, you thinking I should mention something like, "For non-HTTP API purposed (like gRPC) please use Application Access TCP?"

Comment (zmb3) [reply docs/pages/application-access/guides/api-access.mdx:8]: Yes

**Review by zmb3 — APPROVED** [review]

**Pull request merged**